src/bin/stellar-strkey/main.rs

```diff
@@ -24,6 +26,10 @@ struct Root {
 enum Cmd {
     /// Decode strkey
     Decode(decode::Cmd),
+    /// Encode strkey
+    Encode(encode::Cmd),
+    /// Generate the zero strkey
+    Zero(zero::Cmd),
```

Comment:
What is the intended use case for the zero key? The PR description says 'tests', but e.g. the SDK uses arbitrary (not necessarily zero) keys and it's not clear to me which tests would want specifically zero.

Reply:
From time to time I want to get the zero address, so I added the command out of convenience. For example:
https://github.com/search?q=org:stellar%20GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF&type=code

Reply:
Maybe I should also add a generate command that randomly generates strkeys too. I can do that in a follow up.